### PR TITLE
Makefile: Add support for /pybridge scenario

### PR DESCRIPTION
**Pull request by martinpitt**
Build the Python bridge from the cockpit project and install it into the VM so we can run tests against it.

Introduce a new variable `SUBSCRIPTION_MANAGER_BRANCH`, so that it's easier to decouple from other test scenarios.

### CODE REVIEW

**Comment by martinpitt** [comment]
@jelly: So this still looks very poor. Did you do anything else when you were running the tests in https://github.com/cockpit-project/cockpit/pull/18314#issuecomment-1425431843 ?

**Comment by jelly** [comment]
Hmm so https://github.com/cockpit-project/cockpit/commit/f779d872a08ba6cd9832e968ede18700cc81a40c was merged. But it doesn't seem to work. So I guess this needs further investigation.

**Comment by martinpitt** [comment]
The thing that's missing from the pybridge run is "checked out subscription-manager/ ref..". So it's testing against the system-installed sub-man, which seems horribly broken in Fedora 37 -- *that's* what's throwing all these D-Bus errors. I misunderstood the "system" scenario, before we did *not* test fedora-37 against the "system" sub-man, but against git.

**Comment by martinpitt** [comment]
Works now! This is a somewhat temporary thing until the pybridge becomes feature complete and just becomes *the* bridge in the RPM packages. Until then it would be useful to keep this scenario for a few months, to ensure that we don't break it any more. But if that's too gross for you, we can also keep this open as a draft, and just rebase it every now and then to check.

**Comment by martinpitt** [comment]
@ptoscano : Friendly review ping? The first and third hunk should stay, and the big middle one is easy enough to revert once the pybridge becomes the default. Until then, it would give both your and our project more confidence that we don't mess this up again. Thanks!

File: Makefile [hunk 1]
```diff
@@ -176,6 +179,24 @@ $(SUBMAN_TAR): subscription-manager
...
+COCKPIT_PYBRIDGE_REF = main
+COCKPIT_WHEEL = cockpit-0-py3-none-any.whl
+
+$(COCKPIT_WHEEL):
+	# aka: pip wheel git+https://github.com/cockpit-project/cockpit.git@${COCKPIT_PYBRIDGE_REF}
+	rm -rf tmp/pybridge
+	git init tmp/pybridge
+	git -C tmp/pybridge remote add origin https://github.com/cockpit-project/cockpit
+	git -C tmp/pybridge fetch --depth=1 origin ${COCKPIT_PYBRIDGE_REF}
+	git -C tmp/pybridge reset --hard FETCH_HEAD
```
Comment (ptoscano) [inline Makefile:193]: hm am I missing anything, or wouldn't it be easier to simply do:
```
git clone --quiet --depth=1 https://github.com/cockpit-project/cockpit tmp/pybridge
```

**Review by ptoscano — COMMENTED** [review]
Sorry for the late review, lots of things were ongoing in the past, not much time to look at "niceties".

This seems mostly OK, I left only a note about something that IMHO can be simpler. Other than that, can you please rebase it?

Thanks!

File: Makefile (hunk 1 above)
Comment (martinpitt) [reply Makefile]: This was done to support a non-main COCKPIT_PYBRIDGE_REF, for testing against a proposed bridge change. But arbitrary refs  are probably not that important. I simplified it.

**Review by ptoscano — APPROVED** [review]
Thanks!

**Pull request merged**